lib/team.rb

```diff
@@ -44,6 +45,9 @@ def initialize(data)
       link: link
     }
 
+    #Same thing as above?
+    #@team_info = {data}
```

Comment:
I switched this when working on the file converter. At first it was returning a csv file line as the data. Also the underscores between the franchise_id, short_name and team_name would have to get removed for the test to pass. If we want to switch it back to how it was that's alright with me. The main issue of the csv file line as the data has been solved.